style.css

```diff
@@ -49,45 +58,53 @@ body {
   margin-left: 10px;
   margin-bottom: 20px;
   border: 1px solid black;
-  width: 538px;
-  min-height: max-content;
+  width: 475px;
+
+  /* min-height: max-content; */

```

Comment:
- [ ] It is considered best practice to avoid keeping commented codes or unused codes in your work base, as it makes it messy. Kindly remove this unused part of your code. :wrench: